spec/site/tlc/alarm_spec.rb

```diff
@@ -20,7 +20,64 @@
   # the test is recommended.
 
   describe 'Alarm' do
-    before { skip "Don't yet have a reliable way of triggering alarms" }
+        
+    # Validate that a detector logic fault A0302 raises and removed.
+    # 1. Given the site is connected
+    # 2. We trigger an alarm by setting S0003 input 152 to True
+    # 3. Then we should receive an active alarm
+    # 4. We remove an alarm by setting S0003 input 152 to False
+    # 5. Then we should receive an inactive alarm
+
+    specify 'A0302 is raised when a detector logic is faulty', :script, sxl: '>=1.0.7' do |example|
+      Validator::Site.connected do |task,supervisor,site|
+
+        # Alarm is raised by setting S0003 input 152 to True
+        ensure_input 'True', 152 do
```

Comment:
using ensure_input with with a block here is a bit odd, since the function does not take a block as an argument